mmpose/datasets/datasets/hand/interhand2d_dataset.py

```diff
@@ -0,0 +1,269 @@
+
+        Returns:
+            cam_coord (ndarray[N, 3]): 3D joints coordinates
+                in the camera coordinate system
+        """
+        cam_coord = np.dot(R, world_coord - T)
+        return cam_coord
+
+    def _get_db(self):
+        """Load dataset.
+
+        Adapted from 'https://github.com/facebookresearch/InterHand2.6M/'
+                        'blob/master/data/InterHand2.6M/dataset.py'
+        Copyright (c) FaceBook Research, under CC-BY-NC 4.0 license.
+        """
+        with open(self.camera_file, 'r') as f:
+            cameras = json.load(f)
+        with open(self.joint_file, 'r') as f:
+            joints = json.load(f)
+        gt_db = []
+        for img_id in self.img_ids:
+            num_joints = self.ann_info['num_joints']
+
+            ann_id = self.coco.getAnnIds(imgIds=img_id, iscrowd=False)
+            ann = self.coco.loadAnns(ann_id)[0]
+            img = self.coco.loadImgs(img_id)[0]
+
+            capture_id = str(img['capture'])
+            camera_name = img['camera']
+            frame_idx = str(img['frame_idx'])
+            image_file = os.path.join(self.img_prefix, self.id2name[img_id])
+
+            camera_pos, camera_rot = np.array(
+                cameras[capture_id]['campos'][camera_name],
+                dtype=np.float32), np.array(
+                    cameras[capture_id]['camrot'][camera_name],
+                    dtype=np.float32)
+            focal, principal_pt = np.array(
+                cameras[capture_id]['focal'][camera_name],
+                dtype=np.float32), np.array(
+                    cameras[capture_id]['princpt'][camera_name],
+                    dtype=np.float32)
+            joint_world = np.array(
+                joints[capture_id][frame_idx]['world_coord'], dtype=np.float32)
+            joint_cam = self._world2cam_(
+                joint_world.transpose(1, 0), camera_rot,
+                camera_pos.reshape(3, 1)).transpose(1, 0)
+            joint_img = self._cam2pixel_(joint_cam, focal, principal_pt)[:, :2]
+            joint_img = joint_img.reshape(2, -1, 2)
+
+            joint_valid = np.array(
+                ann['joint_valid'], dtype=np.float32).reshape(2, -1)
+            # if root is not valid -> root-relative 3D pose is also not valid.
+            # Therefore, mark all joints as invalid
+            for hand in range(2):
+                joint_valid[hand, :] *= joint_valid[hand][-1]
+
+                if np.sum(joint_valid[hand, :]) > 11:
+                    rec = []
+                    joints_3d = np.zeros((num_joints, 3), dtype=np.float32)
+                    joints_3d_visible = np.zeros((num_joints, 3),
+                                                 dtype=np.float32)
+                    joints_3d[:, :2] = joint_img[hand, :, :]
+                    joints_3d_visible[:, :2] = np.minimum(
+                        1,
+                        np.array(joint_valid[hand, :]).reshape(-1, 1))
```

Comment:
`np.array(joint_valid[hand, :]).reshape(-1, 1)` -> `joint_valid[hand, :].reshape(-1, 1)` since joint_valid is an array already